irobot_benchmark/comms_benchmark/scripts/actions/run_mix_process_benchmarks.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+SCALING_GOVERNOR=$(cat "/sys/devices/system/cpu/cpufreq/policy0/scaling_governor")
+
+# Check if the governor is set to 'performance'
+if [ "$SCALING_GOVERNOR" != "performance" ]; then
+  echo "Warning: CPU scaling governor is not set to 'performance'. Current value is '$SCALING_GOVERNOR'."
+  echo "run: 'echo performance > /sys/devices/system/cpu/cpufreq/policy0/scaling_governor'"
```

Comment:
as part of "making this into a framework", maybe we should by default error out if the wrong governor is set and have a flag like `--any-governor` that bypasses the error and keeps going

Reply:
Now the script handles the setting and resetting of the governor automatically, issuing errors if can't be done (like lacking sudo permissions)